apps/desktop/electron/database/migrations/0000_organic_apocalypse.sql

```diff
@@ -0,0 +1,266 @@
+-- Current sql file was generated after introspecting the database
```

Comment:
The files in this directory were generated by `drizzle-kit pull`. I used that to scaffold out the relations and schema files. We're not using drizzle's migrations yet.